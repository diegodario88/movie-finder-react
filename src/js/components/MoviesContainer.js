
import React, { Component } from 'react'
import http from '../service/http'

class MoviesContainer extends Component {
  constructor(props) {
    super(props)
  }
  componentDidMount() {
    http.http.get(`search/movie?api_key=${http.key}&query=pulp+fiction`).then(res => window.console.log(res))
  }
  render() {
    return (
      <h1>Hello from MoviesContainer</h1>
    )
  }
}

export default MoviesContainer
