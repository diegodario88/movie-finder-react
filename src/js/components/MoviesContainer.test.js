
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MoviesContainer from './MoviesContainer';

import '../enzyme';

describe('MoviesContainer', () => {
  it('renders component properly', () => {
    expect(toJson(shallow(<MoviesContainer />))).toMatchSnapshot();
  });
});
