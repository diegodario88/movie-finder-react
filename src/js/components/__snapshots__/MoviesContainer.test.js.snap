// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesContainer renders component properly 1`] = `
<div>
  <ToastContainer
    autoClose={5000}
    bodyClassName={null}
    className={null}
    closeButton={
      <DefaultCloseButton
        ariaLabel="close"
      />
    }
    closeOnClick={true}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnHover={true}
    position="top-right"
    progressClassName={null}
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
  <MoviesContainer
    errorHandler={[Function]}
  />
</div>
`;
